components/package.json

```diff
@@ -1,12 +1,30 @@
 {
-	"version": "0.1.0",
-	"description": "",
-	"main": "index.js",
+	"name": "components",
+	"private": true,
+	"version": "0.0.0",
 	"type": "module",
+	"main": "src/index",
```

Comment:
will this try to read the `src/index.tsx` file? fairly confident this "main" should point at a regular js file (usually the result of a transpilation process), but perhaps Vite works differently?

Reply:
Ah I see now - yes tsx did work fine but now I've changed to .js and .d.ts files

Reply:
thanks for changing that! at risk of stating the obvious, I'm recommending js over tsx because a shared library shouldn't enforce integrators to use jsx or ts in their builds.